.changeset/forty-bobcats-hope.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro-stylex": major
```

Comment:
```suggestion
"astro-stylex": minor
```
A major release would bring us to `1.0` - a little soon considering stylex itself isn't `1.0`.

Reply:
Apologies. My  mistake.